docs/key-issues/native-dependencies/blas_openmp.md

```diff
@@ -184,6 +184,39 @@ Linux package managers may do something similar
 or they may use a more a more specific way of managing BLAS and LAPACK, such as
 [Fedora using Flexiblas](https://docs.fedoraproject.org/en-US/packaging-guidelines/BLAS_LAPACK/#_user_level_selection).
 
+??? info "BLAS/LAPACK demuxing: FlexiBLAS, libblastrampoline & SciPy's `cython_blas`/`cython_lapack`"
+
+    Supporting multiple BLAS and LAPACK libraries is quite challenging,
+    because of API and ABI incompatibilities between them. In particular:
+
+    - API's using 32-bit (LP64) or 64-bit (ILP64) integers. Typically a
+      BLAS/LAPACK library supports both at the source level, and how the
+      library is built determines which one is used. ILP64 builds may or may
+      not use a symbol suffix - `64_` being the standard one, but that's not
```

Comment:
"common" instead of "standard"?  As a Julia user I'm sympathetic to the use of this suffix, but I'm not sure it's really a formal standard, as you note right after that this isn't a universal convention.

Reply:
Good point, "common" sounds fine. It's nothing formal indeed, there's an OpenBLAS issue somewhere where that was the outcome - IIRC Julia, R and Python OpenBLAS users settled on that.

Reply:
And MKL ships with both non-suffixed and `64_` suffixed symbols in the same shared libraries.